### PR TITLE
Add missing import for 3.3 version

### LINKED ISSUE #382: Fix failures on the 3.3 branch
In preparation of the new 3.3.0 release #324 has been cherry-picked from the master branch onto a new 3.3 branch but this is causing some namespace related failures.

We should fix this so we can make the release.

### PR DESCRIPTION
**Pull request by haringsrob**
Fixes #382

### CODE REVIEW

**Comment by haringsrob** [comment]
Cool had a false positive once, but after restarting all went fine.

CC: @jhedstrom @pfrenssen

File: features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -21,7 +22,6 @@ class FeatureContext implements Context {
    * @beforeNodeCreate
    */
   public static function alterNodeParameters(BeforeNodeCreateScope $scope) {
-    call_user_func('parent::alterNodeParameters', $scope);
```
Comment (pfrenssen) [inline features/bootstrap/FeatureContext.php]: This line should remain or it won't resolve node creation timestamps correctly under Drupal 8. Seems we have no tests that deal with timestamps on D8 :-/

Comment (pfrenssen) [inline features/bootstrap/FeatureContext.php]: Try this:

```
    call_user_func('\Drupal\DrupalExtension\Context\RawDrupalContext::alterNodeParameters', $scope);
```

Comment (haringsrob) [inline features/bootstrap/FeatureContext.php]: Done.

**Comment by pfrenssen** [comment]
Awesome, thanks!

**Pull request merged**